src/app/startbilde/Startbilde.tsx

```diff
@@ -70,13 +70,13 @@ class Startbilde extends React.Component<StartbildeProps> {
 function mapStateToProps(state: AppState, routeProps: RouteComponentProps<{}>): StartbildeStateProps {
     return {
         valgtEnhet: '4100',
-        valgtTemagruppe: state.valgtTemagruppe,
+        valgtTemagruppe: '',
```

Comment:
hvorfor er denne byttet til tom streng?

Reply:
Hele den filen virker uferdig, de har jo også hardkodet valgtEnhet, gadd ikke skrive komponenten for å håndtere forskjellige verdier når jeg ikke vet hvordan den skal fungere i fremtiden.

Reply:
👍